### DISCUSSION TITLE
caps-lock on virtual keyboard

### DISCUSSION THREAD

**Discussion by HT-7** [linuxmint/discussions#651] [Ideas]
The virtual keyboard does not seem to have a caps lock button. Or is it just very well hidden?

**Comment by ye110wd** [comment]
There is none.

I'd like a 105 and or 87 key keyboard. Why should people with accessibility issues be denied key or have to spend an extra 5 seconds entering characters.

I'd also want a windowed keyboard. Having it glued to the bottom of the screen is so annoying.

This is why almost everyone uses Onboard.
